packages/compass-crud/src/components/table-view/cell-editor.tsx

```diff
@@ -483,14 +481,6 @@ class CellEditor
     return (
       <div>
         <span className={this.wrapperStyle()}>
-          <Tooltip
```

Comment:
While looking for things that still use hadron-react-components I noticed that this is (I think) the last Tooltip. But I can't get it to show up. So it seems to not be doing anything anymore? If so I guess we can just remove it. Otherwise I suppose we'll have to replace it.

If it is still used, please tell me how to trigger it ;)

Reply:
The reason why I'm reasonably sure that the tooltip doesn't work is because the corresponding text input doesn't have any of data-tip, data-place or data-for.

There's actually still one left in aggregations that will be replaced shortly: https://github.com/mongodb-js/compass/blob/4b5384b621dc280fa8e616257311124ba2decf9c/packages/compass-aggregations/src/components/stage-editor-toolbar/add-after-stage.jsx#L33-L47

And I realised something in compass schema as well: https://github.com/mongodb-js/compass/blob/4b5384b621dc280fa8e616257311124ba2decf9c/packages/compass-schema/src/components/type/type.jsx#L135-L142

Reply:
We're updating those other two tooltips in this pr: https://github.com/mongodb-js/compass/pull/3792 so if we remove/update that one in crud here (or another pr) we can remove `react-tooltip` 🙌